src/components/widgets/Uenigboks.jsx

```diff
@@ -23,9 +23,9 @@ const Uenigboks = ({ label }) => {
 
     useEffect(() => {
         setInputValue(uenighet.value || '');
-        if (uenighet.value) {
+        if (uenighet.value !== undefined) {
             setChecked(true);
```

Comment:
Vil ikke avkryssingsboksen alltid være krysset av så lenge input-feltet er aktivert? Med andre ord at `uenighet.value` ikke kan endre seg hvis `checked === false`.

Reply:
Dette er for at avkryssingsboksen skal være krysset ved mount. Fjerner man denne sjekken og laster inn siden på nytt vil ikke boksen være krysset av selv om det finnes en uenighet.